internal/kafka/command_topic_list.go

```diff
@@ -10,10 +10,10 @@ import (
 )
 
 type topicOut struct {
-	Name              string `human:"Name" serialized:"name"`
-	IsInternal        bool   `human:"Internal" serialized:"is_internal"`
-	ReplicationFactor int32  `human:"Replication Factor" serialized:"replication_factor"`
-	PartitionCount    int32  `human:"Partition Count" serialized:"partition_count"`
+	Name              string `human:"Name" json:"name" yaml:"name"`
+	IsInternal        bool   `human:"Internal" json:"is_internal" yaml:"is_internal"`
+	ReplicationFactor int32  `human:"Replication Factor" json:"replication_factor" yaml:"replication_factor"`
+	PartitionCount    int32  `human:"Partition Count" json:"partition_count" yaml:"partition_count"`
```

Comment:
Any reason for undoing this?

Reply:
Yeah, I'm embedding this into the combined output struct for the new on-prem `describe`, and `output.SerializedOutput` doesn't convert `serialized` to `json` and `yaml` (yet?).

Reply:
Can we use `output.NewTable()` or `output.NewList()` instead of `output.SerializedOutput()`?

Reply:
Yup. Also I figured out what was causing some `serialized` tags to not display correctly w/ `table`. Turns out you mixing fields that have `serialized` and fields that have `json`/`yaml` doesn't work.